motoko/cert-var/src/cert_var_assets/src/index.js

```diff
@@ -0,0 +1,66 @@
+import { Actor, HttpAgent } from '@dfinity/agent';
+import { Certificate, makeNonceTransform } from '@dfinity/agent';
+import { cert_var } from '../../declarations'
+
+document.getElementById("certifyBtn").addEventListener("click", async () => {
+  const newVal = BigInt(document.getElementById("newValue").value);
+  const _ = await cert_var.set(newVal);
+  const resp = await cert_var.get();
+
+  // to do -- From where to get the port and host, in general?
+  const port = 8000;
+  const agent = await Promise.resolve(new HttpAgent({ host: 'http://127.0.0.1:' + port })).then(
+      async agent => {
+          await agent.fetchRootKey();
+          agent.addTransform(makeNonceTransform());
+          return agent;
+      },
+  );
+  const cert = new Certificate(resp.certificate[0], agent);
```

Comment:
@chenyan-dfinity Can you take a look at this line?

Unlike before, I think I'm now supplying the blob that this constructor expects to get.

However, I get this error now:
```
cbor.js:86 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'byteLength')
    at Module.decode (cbor.js:86)
    at new Certificate (certificate.js:75)
    at HTMLButtonElement.<anonymous> (index.js:19)
```

Using a breakpoint, I can see that the blob exists and is 442 bytes long, for what it's worth.

BTW, when I leave off the agent, I get an error about the default agent not being defined in this context.  I tried to vary that when I wasn't sure about that being cause, but it didn't help.